pom.xml

```diff
@@ -1,358 +1,349 @@
+			<organizationUrl>http://scalecube.io/</organizationUrl>
+		</developer>
+	</developers>
 
-  <distributionManagement>
-    <snapshotRepository>
-      <id>ossrh</id>
-      <url>https://oss.sonatype.org/content/repositories/snapshots</url>
-    </snapshotRepository>
-    <repository>
-      <id>ossrh</id>
-      <url>https://oss.sonatype.org/service/local/staging/deploy/maven2/</url>
-    </repository>
-  </distributionManagement>
+	<distributionManagement>
+		<snapshotRepository>
+			<id>ossrh</id>
+			<url>https://oss.sonatype.org/content/repositories/snapshots</url>
+		</snapshotRepository>
+		<repository>
+			<id>ossrh</id>
+			<url>https://oss.sonatype.org/service/local/staging/deploy/maven2/</url>
+		</repository>
+	</distributionManagement>
 
-  <properties>
-    <argLine>-Xms1024m -Xmx2048m</argLine>
-    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+	<properties>
+		<argLine>-Xms1024m -Xmx2048m</argLine>
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+		<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 
-    <slf4j.version>1.7.7</slf4j.version>
-    <rxjava.version>1.3.0</rxjava.version>
-    <findbugs.version>2.0.3</findbugs.version>
-    <netty.version>4.1.21.Final</netty.version>
-    <socketio.version>2.3.0</socketio.version>
-    <protostuff.version>1.6.0</protostuff.version>
-    <protobuf.version>3.4.0</protobuf.version>
-    <metrics.version>3.1.2</metrics.version>
-    <jackson.version>2.9.0</jackson.version>
-    
-    <maven-gpg-plugin.version>1.6</maven-gpg-plugin.version>
-    <maven-source-plugin.version>3.0.1</maven-source-plugin.version>
-    <maven-javadoc-plugin.version>3.0.1</maven-javadoc-plugin.version>
-    
-    
+		<slf4j.version>1.7.7</slf4j.version>
+		<netty.version>4.1.25.Final</netty.version>
+		<reactor.version>3.1.8.RELEASE</reactor.version>
+		<socketio.version>2.3.0</socketio.version>
+		<protostuff.version>1.6.0</protostuff.version>
+		<protobuf.version>3.4.0</protobuf.version>
+		<metrics.version>3.1.2</metrics.version>
+		<jackson.version>2.9.5</jackson.version>
 
-    <skipStressTests>true</skipStressTests>
-  </properties>
+		<maven-gpg-plugin.version>1.6</maven-gpg-plugin.version>
+		<maven-source-plugin.version>3.0.1</maven-source-plugin.version>
+		<maven-javadoc-plugin.version>3.0.1</maven-javadoc-plugin.version>
+		<skipStressTests>true</skipStressTests>
+	</properties>
 
-  <modules>
-    <module>transport</module>
-    <module>cluster</module>
-    <module>testlib</module>
-    <module>benchmark</module>
-    <module>scalecube-cluster-examples</module>
-    <module>utils</module>
-    <!-- <module>stresstests</module> -->
-  </modules>
+	<modules>
+		<module>transport</module>
+		<module>cluster</module>
+		<module>testlib</module>
+		<module>benchmark</module>
+		<module>scalecube-cluster-examples</module>
```

Comment:
Strange name, everywerhe it's single word, and examples  called as `blabla-bla-examples`. Better to follow already est. naming convention, i.e. single word.